tls/s2n_tls_parameters.h

```diff
@@ -76,12 +76,20 @@
 /* TLS 1.3 hybrid post-quantum definitions are from the proposed reserved range defined
  * in https://tools.ietf.org/html/draft-stebila-tls-hybrid-design. Values for interoperability
  * are defined in https://docs.google.com/spreadsheets/d/12YarzaNv3XQNLnvDsWLlRKwtZFhRrDdWf36YlzwrPeg/edit#gid=0. */
-#define TLS_PQ_KEM_GROUP_ID_X25519_SIKE_P434_R2 0x2F27
-#define TLS_PQ_KEM_GROUP_ID_SECP256R1_SIKE_P434_R2 0x2F1F
-#define TLS_PQ_KEM_GROUP_ID_X25519_BIKE1_L1_R2 0x2F28
-#define TLS_PQ_KEM_GROUP_ID_SECP256R1_BIKE1_L1_R2 0x2F23
-#define TLS_PQ_KEM_GROUP_ID_X25519_KYBER_512_R2 0x2F26
-#define TLS_PQ_KEM_GROUP_ID_SECP256R1_KYBER_512_R2 0x2F0F
+#define TLS_PQ_KEM_GROUP_ID_X25519_SIKE_P434_R2     0x2F27
+#define TLS_PQ_KEM_GROUP_ID_SECP256R1_SIKE_P434_R2  0x2F1F
+#define TLS_PQ_KEM_GROUP_ID_X25519_BIKE1_L1_R2      0x2F28
+#define TLS_PQ_KEM_GROUP_ID_SECP256R1_BIKE1_L1_R2   0x2F23
+#define TLS_PQ_KEM_GROUP_ID_X25519_KYBER_512_R2     0x2F26
+#define TLS_PQ_KEM_GROUP_ID_SECP256R1_KYBER_512_R2  0x2F0F
+
+/* TODO: These are values are placeholders until we confirm with LibOQS */
+#define TLS_PQ_KEM_GROUP_ID_X25519_SIKE_P434_R3     0x2F35
+#define TLS_PQ_KEM_GROUP_ID_SECP256R1_SIKE_P434_R3  0x2F36
```

Comment:
Relating to above, I imagine that the sike_r3 identifiers will be (or at least "can be", or maybe "should be") the same as the sike_r2 identifiers.